static/app/views/eventsV2/table/quickContext.tsx

```diff
@@ -320,23 +372,23 @@ export function QuickContextHoverWrapper(props: ContextProps) {
         skipWrapper
         delay={HOVER_DELAY}
         body={
-          <Wrapper data-test-id="quick-context-hover-body">
-            {isIssueContext(props.contextType) ? (
-              <IssueContext
-                api={api}
-                dataRow={props.dataRow}
-                eventID={props.dataRow.id}
-              />
-            ) : isReleaseContext(props.contextType) && props.organization ? (
-              <ReleaseContext
-                api={api}
-                dataRow={props.dataRow}
-                organization={props.organization}
-              />
-            ) : (
-              <NoContextWrapper>{t('There is no context available.')}</NoContextWrapper>
-            )}
-          </Wrapper>
+          isIssueContext(props.contextType) ? (
+            <IssueContext api={api} dataRow={props.dataRow} eventID={props.dataRow.id} />
+          ) : isReleaseContext(props.contextType) && props.organization ? (
+            <ReleaseContext
+              api={api}
+              dataRow={props.dataRow}
+              organization={props.organization}
+            />
+          ) : isEventContext(props.contextType) && props.organization ? (
+            <EventContext
+              api={api}
+              dataRow={props.dataRow}
+              organization={props.organization}
+            />
+          ) : (
+            <NoContextWrapper>{t('There is no context available.')}</NoContextWrapper>
+          )
```

Comment:
nit: These ternary operators might get harder to read as we add more context types. I think we can probably consolidate the logic here to some kind of map or switch case statement. Not a blocker for this pr but something we can think about for a future pr

Reply:
Yeah will be using switch cases for this. Logged it in the cleanup ticket: https://getsentry.atlassian.net/browse/DD-1170